docs-src/content/getting-started/streaming.md

```diff
@@ -69,4 +69,37 @@ func main() {
 }
 ```
 {{% /tab %}}
+
+{{% tab "Python" %}}
+``` python
+import cubic
+
+serverAddress = '127.0.0.1:2727'
+
+client = cubic.Client(serverAddress)
+
+# use the first available model
+modelResp = client.ListModels()
```

Comment:
I think it'd be helpful to have something here which prints out all of the models, so clients can quickly see how to navigate the modelResp object

Reply:
good point, will do